thirdparty/zlib/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(ZLIB_VER "1.2.11")
 set(ZLIB_MD5 "1c9f62f0778697a09d36121ead88e08e")
 ExternalProject_Add(
     ${PROJECT_NAME}
-    DOWNLOAD_DIR ${KO_DOWNLOAD_DIR}
+    DOWNLOAD_DIR @KO_DOWNLOAD_DIR@
```

Comment:
To clarify, `configure_file()` copies a file and replaces all variables. By using `configure_file()` with the `@ONLY` option regular variables remain intact and are processed as they would normally, but only `@VARIABLE@` is replaced.